### PR TITLE
Check focus state when a key event occurs

### PR DESCRIPTION
**Pull request by JSUYA**
KeyEvent is dispatched only when FlutterView is in focused state.
If it is not focused, it returns false to propagate the event to the parent.

related pr : https://github.com/flutter-tizen/engine/pull/354

### CODE REVIEW

File: embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs [hunk 1]
```diff
@@ -140,6 +140,16 @@ public struct FlutterDesktopEngineProperties
             uint scan_code,
             [MarshalAs(UnmanagedType.U1)]
             bool is_down);
+
+        [DllImport("flutter_tizen.so")]
+        public static extern void FlutterDesktopViewSetFocused(
+                            FlutterDesktopView view,
```
Comment (swift-kim) [inline embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs:146]: Please follow the [formatting convention](https://github.com/flutter-tizen/flutter-tizen/pull/426).

https://github.com/flutter-tizen/flutter-tizen/pull/444/files#diff-225e3c79a0ee53a2065e2001017139ae15e14f249305f64622597e00c3a40f0d

Comment (JSUYA) [reply embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs]: Thanks. done

Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs]: Thanks. Please also consider fixing the `FlutterDesktopViewOnKeyEvent` part.

Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs]: Also `focus` should be either `focused` or `is_focused`.

**Comment by bbrto21** [comment]
@JSUYA Nice work!, could you share a simple example for testing this?

File: embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs (hunk 1 above)
Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs]: > Please also consider fixing the `FlutterDesktopViewOnKeyEvent` part.

Indentation...

**Review by swift-kim — APPROVED** [review]
The engine hash should be updated before merge.

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**